shared/common-adapters/autosize-input.desktop.js

```diff
@@ -106,4 +107,10 @@ const resetStyle = {
   border: 'none',
 }
 
+const placeholderColorCSS = `
+input::placeholder { 
```

Comment:
I think you should add a `className` to the target input, because this style is global and could cause unintended side-effects.